src/resonate/scheduler.py

```diff
@@ -814,8 +936,18 @@ def _process_local_invocation(
         )
 
         if isinstance(invocation.exec_unit, Command):
-            raise NotImplementedError
-        if isinstance(invocation.exec_unit, FnOrCoroutine):
+            assert not isinstance(
+                invocation.exec_unit, CreateDurablePromiseReq
```

Comment:
What do you think of separating `RFICommands` and `LFICommands` to avoid this assert and this usage from the user.

Reply:
If we were to do that we would need to split most datatypes. Like `ExecUnit. I like this already, only `CreateDurablePromiseReq` is a reserved command for rfi. It's easy to check on that